docs/contributing.rst

```diff
@@ -91,55 +88,6 @@ the integration tests suite.
 .. _Go: https://golang.org
 
 
-Vagrant
--------
-
-If you are a Vagrant user, Let's Encrypt comes with a Vagrantfile that
-automates setting up a development environment in an Ubuntu 14.04
-LTS VM. To set it up, simply run ``vagrant up``. The repository is
-synced to ``/vagrant``, so you can get started with:
-
-.. code-block:: shell
-
-  vagrant ssh
-  cd /vagrant
-  ./venv/bin/pip install -r requirements.txt .[dev,docs,testing]
-  sudo ./venv/bin/letsencrypt
-
-Support for other Linux distributions coming soon.
-
-.. note::
-   Unfortunately, Python distutils and, by extension, setup.py and
-   tox, use hard linking quite extensively. Hard linking is not
-   supported by the default sync filesystem in Vagrant. As a result,
-   all actions with these commands are *significantly slower* in
-   Vagrant. One potential fix is to `use NFS`_ (`related issue`_).
-
-.. _use NFS: http://docs.vagrantup.com/v2/synced-folders/nfs.html
-.. _related issue: https://github.com/ClusterHQ/flocker/issues/516
-
-
-Docker
-------
-
-OSX users will probably find it easiest to set up a Docker container for
```

Comment:
Does `letsencrypt` and all of it's tests now work on OS X without Docker? At the very least, integration tests do not work for @jmhodges.